point-of-sale-app/skaffold.yaml

```diff
@@ -41,5 +41,4 @@ deploy:
     - kubernetes-manifests/api-server-lb.yaml
     - kubernetes-manifests/inventory.yaml
     - kubernetes-manifests/payments.yaml
-# [END anthosbaremetal_point_of_sale_app_skaffold_config_backend]
----
+# [END anthosbaremetal_point_of_sale_app_skaffold_config_backend]
```

Comment:
is this pr up to date with main? I thought we removed the `---` in https://github.com/GoogleCloudPlatform/point-of-sale/pull/80

Reply:
Ahhh good catch. I have to pull the changes in.

Reply:
Updated in https://github.com/GoogleCloudPlatform/point-of-sale/pull/82/commits/f3a30a171d641668eb78a766cc33bf6dd2d5f7c6